plugins/woocommerce/tests/e2e-pw/test-data/data.js

```diff
@@ -102,9 +102,9 @@ const storeDetails = {
 			zip: 'Division 1',
 			email: admin.email,
 			country: 'Liberia — Margibi', // corresponding to the text value of the option,
-			countryCode: 'LR',
+			countryCode: 'LR:MA',
 		},
-		expectedNumberOfIndustries: 8, // There are 8 checkboxes on the page (in Liberia), adjust this constant if we change that
+		expectedNumberOfIndustries: 7, // There are 7 checkboxes on the page (in Liberia), adjust this constant if we change that
```

Comment:
I encountered this issue during PR testing so I went ahead and included this minor fix.